code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -559,6 +559,9 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 
 /obj/item/melee/baton/abductor/proc/SleepAttack(mob/living/L,mob/living/user)
 	playsound(src, stun_sound, 50, TRUE, -1)
+	if(L.IsSleeping())
+		L.visible_message("<span class='danger'>[user] tries to induce sleep in [L] with [src], but [L.p_they()] is already asleep!</span>")
```

Comment:
```suggestion
		L.visible_message("<span class='danger'>[user] tries to induce sleep in [L] with [src], but [L.p_they()] [L.p_is()] already asleep!</span>")
```

Otherwise you're going to get "they is already asleep!" for gender neutral mobs.